### PR TITLE
Make 'Apply' journey bookends TA specific

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

This PR updates Approved Premises text that we've inherited in our application journey to be Temporary Accommodation-specific. The goal is not for these screens to be final, but for them to be not obviously wrong for TA.

We also take this as an opportunity to remove the AP-pattern of passing page titles into the template from the controller, which is not a pattern followed in TA.

### Before
localhost_3000_applications_58f6d529-9631-4b1a-8055-1da04bfcbc02_submission
localhost_3000_applications_58f6d529-9631-4b1a-8055-1da04bfcbc02
localhost_3000_applications_people_C805002_select-offence
localhost_3000_applications_new__csrf=1OL9dKnO-ddZQv-02oUI4wxQ7Xn9Y_hrMwbY
localhost_3000_applications_start
localhost_3000_applications_

### After

localhost_3000_applications_58f6d529-9631-4b1a-8055-1da04bfcbc02_submission
localhost_3000_applications_58f6d529-9631-4b1a-8055-1da04bfcbc02
localhost_3000_applications_people_C805002_select-offence
localhost_3000_applications_new__csrf=Cy0JTLKd-1QtQCyLMrpNj25Oym9CxJalCZao
localhost_3000_applications_start (1)
localhost_3000_applications_

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Make 'Apply' journey bookends TA specific](https://trello.com/c/Adkyy6Zk/862-make-apply-journey-bookends-ta-specific)

File: server/views/applications/confirm.njk [hunk 1]
```diff
@@ -4,40 +4,33 @@
...
-            <p>
-                <a href="https://www.gov.uk/service-manual/user-centred-design/resources/patterns/feedback-pages.html">What did you think of this service?</a> (takes 30 seconds)</p>
-
-        </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukPanel({
+        titleText: "Referral complete"
+      }) }}
+      <p>We’ve sent you a confirmation email.</p>
```
Comment (tahb) [inline server/views/applications/confirm.njk]: Is this true? I wasn't aware that CAS1 were sending confirmation emails. As far as I know the Notify integration has yet to be merged into the API yet?

**Review by tahb — APPROVED** [review]
LGTM apart from question about emailing confirmations which may need to be removed?

Are we removing dynamic page headings because we want all our browser page titles to be the same, is that the only change?

File: server/views/applications/confirm.njk (hunk 1 above)
Comment (bryangaledxw) [reply server/views/applications/confirm.njk]: Oh, no, not true - pretty sure this is aspirational. Will remove so no-one gets their hopes up!

Comment (bryangaledxw) [reply server/views/applications/confirm.njk]: The dynamic page headings - it's just not really a pattern we've followed in CAS3. You either have the heading in the njk, or you have it in the controller, and on CAS3 we've gone for njk. Just wanted to keep things consistent while I was revisiting these files

**Pull request merged**